CoreScriptsRoot/ServerStarterScript.lua

```diff
@@ -35,7 +35,7 @@ end
 --[[ Remote Events ]]--
 local RemoteEvent_SetDialogInUse = Instance.new("RemoteEvent")
 RemoteEvent_SetDialogInUse.Name = "SetDialogInUse"
-RemoteEvent_SetDialogInUse.Parent = RobloxReplicatedStorage 
+RemoteEvent_SetDialogInUse.Parent = RobloxReplicatedStorage
```

Comment:
At some point in time we need to communicate on dev forums + publically that new chat will create remoteevents in this folder. I know some games sweep through workspace checking for unexpected things?